components/brave_wallet_ui/components/desktop/popup-modals/sell-asset-modal/sell-asset-modal.tsx

```diff
@@ -82,6 +91,7 @@ export const SellAssetModal = (props: Props) => {
   }, [])
 
   // Computed
+  const generatedAddress = useReceiveAddressQuery(account?.accountId)
```

Comment:
But we are selling BTC for dollars, why we need new receive address? :)
How that should work?

Reply:
`userAddress=<address>` is required as a param from `Ramp` else we get an error on there end. I believe it's so they can watch what address funds are sent from to accurately pay out to the right user. But maybe that should be the `PublicKey` for `Bitcoin` off-ramping? Asking the team in our shared slack channel!

![Screenshot 11](https://github.com/brave/brave-core/assets/40611140/d3c98391-1934-421e-adb9-73e9905bf0a5)